JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -265,7 +265,9 @@ - (void)viewDidLoad
 {
     [super viewDidLoad];
 
-    [[[self class] nib] instantiateWithOwner:self options:nil];
+    // OWS: We don't want to load the nib twice - this has a number of
+    // undesirable effects, such as instantiating two collection views. 
+//    [[[self class] nib] instantiateWithOwner:self options:nil];
```

Comment:
We really need to test - this could have unanticipated consequences - but we definitely don't want to instantiate the nib twice.  It's wasteful and you end up in a situation where two instances of the collection view (and possibly input views?) exist.

Reply:
This seems really weird. Any clue as to why it was happening in the first place?

Reply:
Yeah, they were doing it to allow users to override the "nib" method and use a custom nib, but this isn't a good way to do that, since it has these negative side effects.